google/resource_container_cluster.go

```diff
@@ -274,6 +274,11 @@ func resourceContainerCluster() *schema.Resource {
 										Type:     schema.TypeBool,
 										Required: true,
 									},
+									"load_balancer_type": {
+										Type:         schema.TypeString,
+										ValidateFunc: validation.StringInSlice([]string{"internal"}, true),
```

Comment:
I believe generally we pass the API accepted values through verbatim. Whenever additional settings come around, do you have an idea if they will follow the same naming convention?

I'd rather see `LOAD_BALANCER_TYPE_INTERNAL` here, or at the very least forcing all caps `INTERNAL` (for reasons below)

Reply:
Done. Switched to `LOAD_BALANCER_TYPE_INTERNAL`